test/compress/evaluate.js

```diff
@@ -95,6 +101,12 @@ unsafe: {
             2
         );
         console.log(
+            ({a:{b:1},a:1}) + 1,
+            2,
+            ({a:{b:1},a:1}).b + 1,
+            1..b + 1
```

Comment:
@kzc indeed this is getting incomprehensible - I'm splitting them out as we speak 😅 

`1..b` is correct as far as I can tell - one dot would cause syntax error as it confuses with numeral literals.